<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ecualizarBoton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>370</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ecualizar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numVecinos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_filtro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagenF">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformaciones de color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filtro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtros:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonTrans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RGB -&gt; Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; Opponent</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; YCbCr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; YUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; YIQ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; HSI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; CIE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB -&gt; I1I2I3</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboFiltros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Promedio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaussiano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Maximo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Minimo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mediana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nagao-Matsuyama</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sigma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prewitt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Roberts</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="botonAbrir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vecinos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de vecinos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcularHistogramaButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>370</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="originalImagen">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagen original</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_constraste">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Correccion Gamma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stretching</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="boton_contraste">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vecinos_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor de Gamma:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valorGamma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>671</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcesamiento_de_imagenes">
    <property name="title">
     <string>Procesamiento de imagenes</string>
    </property>
   </widget>
   <addaction name="menuProcesamiento_de_imagenes"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
